### ISSUE TITLE
Cannot rename bookmark entry with F2 because focus is given to fileview after click

### ISSUE THREAD

**Issue by matzipan** [elementary/files#51]
When I click a bookmark item in sidebar (a custom bookmark that I have created myself, not the default ones), I can't seem to be able to trigger rename with F2 right after I have clicked on it. 

I have to hover my mouse over the file view and then back on the sidebar and press f2 to do this. @jeremypw  was saying this is because the focus is given to the file view after clicking on a bookmark entry to make Files friendly to keyboard users. 

Should the focus be hijacked and given to the file view even though the user is oblivious to it? Why not better keep the focus where it is and give the user a key to press to shift focus to the file view? 

I am unable to check now how mac OS or Windows handle this.

**Comment by jeremypw** [comment]
Pressing the tab key does shift focus between sidebar and view.  We need the opinion of a keyboard only user or accessibility expert really.

**Comment by jeremypw** [comment]
This also raised the question of whether the UI should more clearly indicate the which widget as keyboard focus.

**Comment by jeremypw** [comment]
I think a better solution would be to have a special shortcut for renaming the current bookmark e.g. `<Shift>F2`.  This would suit both mouse and keyboard (and mixed) users and be easier to implement.

**Comment by jeremypw** [comment]
After further thought I decided it would be better to have a "focus-sidebar" action as this would allow access to all sidebar keyboard actions, not just renaming.  So to rename a bookmark when the view is focused without using the mouse you would press `<Ctrl>Left`, select the required bookmark if not already selected using the Up and Down keys, then press `F2`

**Comment by jeremypw** [comment]
Closing as there is now a keyboard shortcut for focusing the sidebar.
